x-pack/plugins/lists/public/exceptions/api.test.ts

```diff
@@ -40,188 +35,294 @@ const mockKibanaHttpService = ((createKibanaCoreStartMock() as unknown) as jest.
 );
 
 describe('Exceptions Lists API', () => {
-  describe('addExceptionList', () => {
+  describe('#addExceptionList', () => {
     beforeEach(() => {
       fetchMock.mockClear();
       fetchMock.mockResolvedValue(mockExceptionList);
     });
 
-    test('check parameter url, body', async () => {
-      await addExceptionList({
+    test('it uses POST when "list.id" does not exist', async () => {
+      const payload = { ...mockExceptionList };
```

Comment:
The new mocks have cleaned up these tests quite a bit! Thanks @yctercero  & @FrankHassanabad!